VISION.md

```diff
@@ -0,0 +1,38 @@
+
+## Stakeholders / Collaborators
+
+The docs project supports and collaborates with the other NUnit projects and their respective leads -- for reference, those are currently:
+
+| Project | Lead |
+| ------- | ---- |
+| Framework | @rprouse and @jnm2 |
+| Console and Engine | @ChrisMaddock |
+| VS Adapters and Test Generator | @OsirisTerje|
+| NUnit Analyzers | @mikkelbu |
+| Xamarin Runners | @rprouse |
+
+## Values: how we aim to operate the docs project
+
+* **We aim for a culture of proactive collaboration over rigid rules and process.** Our goal is to further our community's knowledge; we won't get hung up on the process if everyone's working toward that end goal.
+* **We aim to make it easy to get involved.** Newcomers will be welcome, and the ability to contribute should be clear.
+* **We aim for progress over consensus.** We'll sometimes merge changes before everyone agrees, in order to keep things moving forward. That also means keeping an open mind about reverting things. Docs are more malleable than code; we'll try to use that to our advantage.
+* **We respect the project teams and their immense contributions.** The docs don't exist without other project leads and members or the core team. Their opinion matters and is always welcome.
+* **We welcome suggestions and contributions.** Whenever possible, we will work to support contributions to the docs. And we consider the raising of issues and suggestions to be a form of contribution.
+* **We aim to broadcast our intentions, radiate information, and work in public.** It should be clear what's in the roadmap for the docs, and we will aim to broadcast the changes we'll make before we take them.
+
+These values will sometimes conflict, and we'll feel it out and talk it out when they do.
+
+## Who makes changes to the docs
+
+Given that there are many projects that feed into the docs and their members and leads work hard to maintain things, our goal is to be able to modify the docs without making it harder for project team members.
+
+* Any change has to pass the docs build process, linters, etc. before being merged. That process will continue to evolve to ensure the docs have a great build & release story.
+* Typos, small content corrections, stylistic consistency, grammatical changes, and clear bug fixes are places where docs team members can make & merge changes without project member approval.
+* In general, we will ask everyone to aim to create an issue before submitting non-routine changes for consideration. This allows space for discussion ahead of changes.
+* We'll attempt to alert project teams or members to changes in their area in case they want to offer their insights. If no project team members respond within 48 hours, the docs team will work with the PR as they see fit (waiting on input, or merging). Should a project team have a concern after a merge, we'll remember our overall values and work with them to resolve any concerns as a high priority.
```

Comment:
This wording seems to add a requirement for every project to respond to every PR within 48 hours, although that may not be what you intend. I think that's a great goal for any project. However...

1. It's not what happens now.
2. It may not be what each project wants to commit to.

Since the "Typos, small content corrections, etc." bullet point already took care of the easy stuff, this point is the one that deals with the hard stuff. It could easily take a few days just to understand whether a major rewrite of certain parts of the docs (e.g. parallel execution of tests) is even correct, without even considering how good the writing is.

Bottom line, I think that certain __kinds__ of changes as well as very broad changes will probably require some kind of approval of the project team. (After all, there are no tests to run to tell us if the docs are correct) 

That said, I'd be happy with just stating that there are some changes that must go through the project team and leaving it up to you to recognize those cases when they come up.

Reply:
@CharliePoole that makes sense to me. You're correct that my intention was to say "if a project doesn't respond within 48 hours, it's left up to the discretion of the docs team" -- meaning, we might move forward, but are going to apply a lot of judgment on that.

I like the way you're approaching that phrasing and I believe I'll adopt that. 👍

Reply:
✅ Changes made to reflect this.